app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderHintViewModel.kt

```diff
@@ -4,30 +4,61 @@
 
 package org.mozilla.tv.firefox.webrender
 
+import androidx.lifecycle.ViewModel
 import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+import io.reactivex.subjects.Subject
+import org.mozilla.tv.firefox.R
 import org.mozilla.tv.firefox.hint.Hint
+import org.mozilla.tv.firefox.hint.HintViewModel
 import org.mozilla.tv.firefox.navigationoverlay.OverlayHintViewModel
 import org.mozilla.tv.firefox.session.SessionRepo
+import org.mozilla.tv.firefox.utils.URLs
+
+private val OPEN_MENU_HINT =  Hint(
+        R.string.hint_press_menu_to_open_overlay,
+        R.string.hardware_button_a11y_menu,
+        R.drawable.hardware_remote_menu
+)
+
+private enum class Event(val display: Boolean) {
```

Comment:
Rename, bc Event is too general? HintEvent isn't quite right, though these are related to hints.

Reply:
I called this `Event` while it was private, but eventually moved it, made it public, and renamed it to `CursorEvent`.  Does that seem ok to you?